src/main/webapp/script.js

```diff
@@ -82,3 +84,19 @@ function createButton(text) {
 function clearElm(elementID) {
     document.getElementById(elementID).innerHTML = "";
 }
+
+function userResult(){
+   
+    clearElm(QUIZ_ELEMENT_ID);
+    const quiz = document.getElementById(QUIZ_ELEMENT_ID);
+    quiz.appendChild(createParagraphElement("Based On Your Results You Should visit..."));
+   
+    getHeader(this);
```

Comment:
Based on the name of the function, I'd expect this to return the header. Maybe"displayResults" instead?